truss/templates/shared/logging.py

```diff
@@ -1,63 +1,95 @@
+import json
 import logging
 import sys
+from datetime import datetime
+from enum import Enum
 
+import loguru
 from pythonjsonlogger import jsonlogger
 
 LEVEL: int = logging.INFO
 
-JSON_LOG_HANDLER = logging.StreamHandler(stream=sys.stdout)
-JSON_LOG_HANDLER.set_name("json_logger_handler")
-JSON_LOG_HANDLER.setLevel(LEVEL)
-JSON_LOG_HANDLER.setFormatter(
-    jsonlogger.JsonFormatter("%(asctime)s %(levelname)s %(message)s")
-)
+
+class Lifecycle(Enum):
+    LOAD = "MODEL_LOAD"
+    STARTUP = "MODEL_STARTUP"
+    REQUEST = "REQUEST"
+
+
+def serialize(record):
+    dt = datetime.fromtimestamp(record["time"].timestamp())
+    formatted_time = dt.strftime("%Y-%m-%d %H:%M:%S,%f")[:-3]
+
+    if record["message"][0] == "{":
+        return record["message"]
```

Comment:
Why is this special case needed? Does this mean that all `print` statements that begin with a curly brace won't get a request id attached? Seems a bit brittle

Reply:
Might not need this line anymore after disabling uvicorn logs. Good catch.